### ISSUE TITLE
77.7.8 Has the following error: 

### ISSUE THREAD

**Issue by Sprungles** [Tobi823/ffupdater#355]
Stacktrace:
```
java.lang.IllegalArgumentException: Failed requirement.
	at de.marmaro.krt.ffupdater.installer.ApkChecker$Companion.throwIfApkFileIsNoValidZipFile(ApkChecker.kt:17)
	at de.marmaro.krt.ffupdater.DownloadActivity.startDownload(DownloadActivity.kt:330)
	at de.marmaro.krt.ffupdater.DownloadActivity.access$startDownload(DownloadActivity.kt:58)
	at de.marmaro.krt.ffupdater.DownloadActivity$startDownload$1.invokeSuspend(DownloadActivity.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7872)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@60638b5, Dispatchers.Main]
Timestamp: 2023-02-28T13:22:25.933

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.8 (134) release |
| Device | PH-1 (lineage_mata, mata, Mata) |
| Manufacturer | essential (Essential Products) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | ODS-PROD, nolenjohnson, release-keys, 1675913280000 |

**Comment by Tobi823** [comment]
Can you delete the cache and storage of the app?

**Comment by Tobi823** [comment]
Should be fixed in 77.7.9

**Comment by Sprungles** [comment]
Absolutely! Wipeing the cache and storage did it! Thank you!
